packages/react-sketch-canvas/src/types/canvas.ts

```diff
@@ -69,3 +69,7 @@ export interface CanvasPath {
    */
   readonly endTimestamp?: number;
 }
+
+export interface TouchExtends extends Touch {
+  touchType: "direct" | " stylus";
```

Comment:
`" stylus"` did you add the whitespace intentionally?

Reply:
Done